From 0d22b4ab02ad516fa628c523e0c5597c6978467b Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Wed, 22 Aug 2018 17:10:55 +0000
Subject: [PATCH 5/8] add TRAP_UNK si_code to signal.h from linux v4.18

used for undiagnosed trap exceptions where linux previously set si_code to 0.
new in linux commit db78e6a0a6f9f7d7277965600eeb1a5b3a6f55a8
---
 include/signal.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/signal.h b/include/signal.h
index a4f85cca..5c48cb83 100644
--- a/include/signal.h
+++ b/include/signal.h
@@ -233,6 +233,7 @@ void (*sigset(int, void (*)(int)))(int);
 #define TRAP_TRACE 2
 #define TRAP_BRANCH 3
 #define TRAP_HWBKPT 4
+#define TRAP_UNK 5
 #define POLL_IN 1
 #define POLL_OUT 2
 #define POLL_MSG 3
-- 
2.17.1

